body-tracking-samples/sample_unity_bodytracking/MoveLibraryFiles.bat

```diff
@@ -1,26 +1,32 @@
 if not exist Assets\Plugins mkdir Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.1.0.1\lib\netstandard2.0\Microsoft.Azure.Kinect.BodyTracking.dll Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.1.0.1\lib\netstandard2.0\Microsoft.Azure.Kinect.BodyTracking.pdb Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.1.0.1\lib\netstandard2.0\Microsoft.Azure.Kinect.BodyTracking.deps.json Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.1.0.1\lib\netstandard2.0\Microsoft.Azure.Kinect.BodyTracking.xml Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.Sensor.1.3.0\lib\netstandard2.0\Microsoft.Azure.Kinect.Sensor.dll Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.Sensor.1.3.0\lib\netstandard2.0\Microsoft.Azure.Kinect.Sensor.pdb Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.Sensor.1.3.0\lib\netstandard2.0\Microsoft.Azure.Kinect.Sensor.deps.json Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.Sensor.1.3.0\lib\netstandard2.0\Microsoft.Azure.Kinect.Sensor.xml Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.Dependencies.0.9.1\lib\native\amd64\release\cublas64_100.dll Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.Dependencies.0.9.1\lib\native\amd64\release\cudart64_100.dll Assets\Plugins
-copy packages\Microsoft.Azure.Kinect.BodyTracking.Dependencies.0.9.1\lib\native\amd64\release\vcomp140.dll Assets\Plugins
+set BODY_TRACKING_TOOLS_PATH="C:\Program Files\Azure Kinect Body Tracking SDK\tools\"
```

Comment:
Is installing BT SDK at C drive a prerequisite for using the sample code?
I know you copy them from there because BT 1.1 doesn't provide nuget packages.

Reply:
+ @qm13 who is making a plan for the similar case, too.